source/User_Guide/Account_Management/reset_username_password.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+weight: 50
+title: Resetting your account credentials
+navigation:
+  show: true
+seo:
+  title: Resetting your account credentials
+  override: true
+  description: Steps for resetting your username and/or password using the SendGrid UI
+
+  
+---
+
+- [Changing your password](#-Changing-your-password)
+    - [Changing your password through the SendGrid UI](#-Changing-your-password-through-the-SendGrid-UI)
+    - [Changing your password via Forgot Password](#-Changing-your-password-via-Forgot-Password)
+- [Changing your username](#-Changing-your-username)
+
+{% anchor h2 %}
+Changing your password
+{% endanchor %}
+
+There are two ways to change your password. Through the SendGrid UI and through the Forgot Password page. For more information on password requirements and security see the [password documentation](https://sendgrid.com/docs/Classroom/Basics/Security/password.html). 
```

Comment:
~There are two ways to change your password. Through the SendGrid UI and through the Forgot Password page.~

If you already know your password, you can change your SendGrid password by editing your [Account Details](https://app.sendgrid.com/settings/account) in the UI. If you've forgotten your password, you can reset it on the [Reset password page](https://app.sendgrid.com/forgot_password).